content/kubeone/master/architecture/requirements/provider_support_matrix/kubevirt/kubevirt.en.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "KubeVirt"
+date = 2021-02-01T14:46:15+02:00
+weight = 7
+
++++
+
+## KubeVirt
```

Comment:
We don't officially support KubeVirt. Does it make sense to have this doc?

Reply:
Its referenced in the KKP docs though

Reply:
KKP does officially support KubeVirt, but that's not the case for KubeOne. I'm worried that if someone sees this doc, they might think we support KubeVirt. I'm not confident in showcasing something that we never tested before.